server/pension/models.py

```diff
@@ -42,8 +42,9 @@ class Fund(models.Model):
 
 class Instrument(models.Model):
     label = models.CharField(max_length=200)
-    instrument_type = models.IntegerField(choices=InstrumentType.choices)
-    instrument_id = models.IntegerField
+    # instrument_type = models.IntegerField(choices=InstrumentType.choices)
```

Comment:
I am having trouble getting the ID of the choice by entering the string from the data in the table. This is probably because the values are in Hebrew.
I created a function that does it in the model (can be moved to the import_csv.py).
    def get_choice(self, choice_str):
        i=0
        for i in range(len(self.choices)):
            if InstrumentType.choices[i][1] == choice_str:
                return self.choices[i][0]
            i = i+1

The function works fine when using a direct Hebrew string as input:
it = InstrumentType.get_choice(InstrumentType, 'ניירות ערך סחירים')
returns 1
but when called with a value from the table it returns None, possibly because I needed to encode when opening the file.
it = InstrumentType.get_choice(InstrumentType, row[5])
returns None

I tried all kinds of encoding decoding but no use.
Any ideas?